src/WebJobs.Extensions.DurableTask/WebJobs.Extensions.DurableTask.csproj

```diff
@@ -45,7 +45,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Azure.DurableTask.AzureStorage" Version="1.6.3" />
+    <PackageReference Include="Microsoft.Azure.DurableTask.AzureStorage" Version="1.6.4" />

```

Comment:
Is the intention to update to the newest version, 1.6.5?

Reply:
Good call!